File: mvn-advent-of-code/src/test/java/advent/twenty_eighteen/Day18SettlersOfTheNorthPoleTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day18SettlersOfTheNorthPoleTest {
    @Test
    public void checkReal() {
        Day18SettlersOfTheNorthPole im = new Day18SettlersOfTheNorthPole("/twenty_eighteen/Day18-SettlersOfTheNorthPole-input.txt");

        assertEquals(588436, im.getPart1Answer().longValue());
        assertEquals(195290, im.getPart2Answer().longValue());
    }

    @Test
    public void checkExample1() {
        Day18SettlersOfTheNorthPole im = new Day18SettlersOfTheNorthPole("/twenty_eighteen/Day18-SettlersOfTheNorthPole-example#1.txt");

        assertEquals(1147, im.getPart1Answer().longValue());
        assertEquals(0, im.getPart2Answer().longValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day18SettlersOfTheNorthPoleTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day18SettlersOfTheNorthPoleTest  extends FlatSpec with Matchers {
  "2018 Day 18 - Input" should "calculate answers" in {
    val dp = new Day18SettlersOfTheNorthPole("/twenty_eighteen/Day18-SettlersOfTheNorthPole-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day18SettlersOfTheNorthPole("/twenty_eighteen/Day18-SettlersOfTheNorthPole-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
